### PR TITLE
[pt-br] Update tutorials/kubernetes-basics/explore/explore-intro.html

### PR DESCRIPTION
**Pull request by stormqueen1990**
### Changes
* Update the explore-intro tutorial in Brazilian Portuguese to remove the Katacoda references.

### CODE REVIEW

File: content/pt-br/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 1]
```diff
@@ -16,124 +20,324 @@
...
 
-        <div class="row">
-            <div class="col-md-8">
-                <h2>Nós (Nodes)</h2>
-                <p>Um Pod sempre será executando em um <b>Nó</b>. Um Nó é uma máquina de processamento em um cluster Kubernetes e pode ser uma máquina física ou virtual. Cada Nó é gerenciado pelo Control Plane. Um Nó pode possuir múltiplos Pods e o Control Plane do Kubernetes gerencia automaticamente o agendamento dos Pods nos nós do cluster. Para o agendamento automático dos Pods, o Control Plane leva em consideração os recursos disponíveis em cada Nó.</p>
+      <div class="row">
+        <div class="col-md-8">
+          <h2>Nós</h2>
+          <p>
+            Um Pod sempre será executando em um <b>Nó</b>. Um Nó é uma
```
Comment (jcjesus) [inline content/pt-br/docs/tutorials/kubernetes-basics/explore/explore-intro.html:112]: Hi @stormqueen1990, please, can you fix this typo in that word "executando" and change to "executado".
Thanks

File: content/pt-br/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 2]
```diff
@@ -16,124 +20,324 @@
...
+          <p><code><b>kubectl proxy</b></code></p>
+          <p>
+            Agora iremos novamente obter o nome do Pod e obter informações do Pod
+            diretamente através do proxy. Para obter o nome do Pod e armazená-lo
+            na variável de ambiente <tt>POD_NAME</tt>, utilize o comando:
+          </p>
+          <p>
+            <code>
+              <b>export POD_NAME="$(kubectl get pods -o go-template --template
+                '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')"</b>
```
Comment (jcjesus) [inline content/pt-br/docs/tutorials/kubernetes-basics/explore/explore-intro.html:261]: Hi, in the line 261 there is an white space between "range" and ".items", please, may you fix these?
Thanks

**Review by jcjesus — CHANGES_REQUESTED** [review]
Hi @stormqueen1990, thanks for your contribution with portuguese community speakers!
Please, can you consider thoses simple two request changes?
Thanks

File: content/pt-br/docs/tutorials/kubernetes-basics/explore/explore-intro.html (hunk 2 above)
Comment (stormqueen1990) [reply content/pt-br/docs/tutorials/kubernetes-basics/explore/explore-intro.html:261]: Hi there, @jcjesus! Could you please point out as a suggestion here in GitHub which whitespace you're referring to?

If you mean `{{range .items}}` should be `{{range.items}}`, while that would still work, our Portuguese version would be diverging from the English version (and this command works with the whitespace present): https://github.com/kubernetes/website/blob/13a9d24f778372fece6b686300c6bbcbcdd8a6ed/content/en/docs/tutorials/kubernetes-basics/explore/explore-intro.html#L154

File: content/pt-br/docs/tutorials/kubernetes-basics/explore/explore-intro.html (hunk 1 above)
Comment (stormqueen1990) [reply content/pt-br/docs/tutorials/kubernetes-basics/explore/explore-intro.html:112]: Just fixed this!

**Review by jcjesus — APPROVED** [review]
Good job @stormqueen1990 
Thanks

**Pull request merged**